src/explore-education-statistics-common/src/modules/charts/components/CustomTooltip.tsx

```diff
@@ -60,8 +64,8 @@ const CustomTooltip = ({
 
                     <strong>
                       {formatPretty(
-                        item.value.toString(),
-                        item.unit,
+                        (item.value as number | string).toString(),
```

Comment:
The value can potentially be undefined or , so we shouldn't just ignore this. We should wrap this with an appropriate undefined check:

```tsx
{typeof item.value !== 'undefined' && (
  <strong>{formatPretty(item.value, ...)}</strong>
)}
```